package/CHANGELOG

```diff
@@ -15,11 +15,12 @@ The rules for this file:
 ------------------------------------------------------------------------------
 ??/??/?? IAlibay, BFedder, inomag, Agorfa, aya9aladdin, shudipto-amin, cbouy,
          HenokB, umak1106, tamandeeps, Mrqeoqqt, megosato, AnirG, rishu235,
-         mtiberti, manishsaini6421
+         mtiberti, manishsaini6421, Sukeerti1
 
  * 2.2.0
 
 Fixes
+  * Updated Elementwise Comparison. (Issue #3535, PR #3619)
```

Comment:
This needs to be a bit more explicit. Something in the lines of "Fix Numpy deprecation warnings about elementwise comparisons."